tests/yam/agama/auto/auto.pm

```diff
@@ -0,0 +1,23 @@
+## Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: First installation using D-Installer current CLI (only for development purpose)
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use base 'y2_installbase';
+use strict;
+use warnings;
+
+use testapi;
+
+
+sub run {
+
+    assert_screen('alp-installer-ui', 120);
```

Comment:
we shouldn't call it "installer", The Installer is the sle/opensuse one, this has a unique name, Agama,  I would just match "Agama" text on top:
https://openqa.opensuse.org/tests/3237762#step/auto/1